wg-naming/recommendations/002-blacklist-whitelist-allowlist-denylist.md

```diff
@@ -0,0 +1,95 @@
+
+**Status:** Accepted
+
+## Suggested Alternatives 
+
+Replacements: 
+- `allowlist/denylist`
+
+## Context
+
+The underlying assumption of the whitelist/blacklist metaphor is that white = good and black = bad. Because colors in and of themselves have no predetermined meaning, any meaning we assign to them is cultural: for example, the color red in many Southeast Asian countries is lucky, and is often associated with events like marriages, whereas the color white carries the same connotations in many European countries. In the case of whitelist/blacklist, the terms originate in the publishing industry – one dominated by America and England, two countries which participated in slavery and which grapple with their racist legacies to this day.
+
+From a technical communication perspective, using whitelist/blacklist as a naming convention applies metaphor (and, in turn, unintended meaning) when it isn’t needed. More directly descriptive words like allowlist/denylist would enhance understanding. Allowlist/denylist, or simply using allowed/denied as an entity prefix has the added benefit of being easily translatable to other human languages.
+
+
+This term measures up against the evaluation framework as follows:
+
+**First order concerns**: 
+
+In short: is the term overt or identity specific? **Not quite.**
+
+* Is the term overtly racist? **Maybe**. See [this article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/) for more. 
+* Is the term overtly sexist, transphobic, or pejorative about a gender identity? **No**
+* Is the term overtly ableist, or pejorative to neurodiverse or disabled people? **No**
+* Is the term overtly homophobic? **No**
+
+
+**Second order concerns**:
+
+In short: is the term ambigously harmful, or is it harmful but not to a specific identity? **Yes**. Once again, see [this article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/)) 
+
+In particular, see the following pull-quote:
+
+> In this context, it is worth examining the origins of the term “blacklist” from the Douglas Harper Etymology Dictionary, which states that its origin and history is:
+
+ > >  n.
+
+ > >  also black-list, black list, “list of persons who have incurred suspicion,” 1610s, from black (adj.), here indicative of disgrace, censure, punishment (attested from 1590s, in black book) + list (n.). Specifically of employers’ list of workers considered troublesome (usually for union activity) is from 1888. As a verb, from 1718. Related: Blacklisted; blacklisting. [32]
+
+> It is notable that the first recorded use of the term occurs at the time of mass enslavement and forced deportation of Africans to work in European-held colonies in the Americas.
+
+While it is not directly attacking specific identities, in other context it does have negative connotations.
+
+* Is the term violent? **Partially**
+* Is the term milataristic? **No**
+
+**Third-order concerns**
+
+* Is the term evocative instead of descriptive? **Partially** 
```

Comment:
I think it's wholly evocative: nothing about blacklist/whitelist is inherently or literally black or white.
```suggestion
* Is the term evocative instead of descriptive? **Yes** 
```